src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -226,12 +229,25 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
         artifact_prefix=reproducers_dir,
         extra_env=options.extra_env)
 
-    if (not environment.get_value('USE_MINIJAIL') and
-        fuzz_result.return_code == constants.LIBFUZZER_ERROR_EXITCODE):
+    project_qualified_fuzzer_name = (
```

Comment:
nit: can you put this in its own method? it's starting to look a little cluttered here :)

Reply:
Done!